packages/eslint-config-airbnb-base/rules/errors.js

```diff
@@ -131,13 +136,19 @@ module.exports = {
     // disallow negating the left operand of relational operators
     // https://eslint.org/docs/rules/no-unsafe-negation
     'no-unsafe-negation': 'error',
+
+    // Disallow useless backreferences in regular expressions
+    // https://eslint.org/docs/rules/no-useless-backreference
+    // TODO: enable, semver-minor, once eslint v7 is required (which is major)
+    'no-useless-backreference': 'off',
+
     // disallow negation of the left operand of an in expression
     // deprecated in favor of no-unsafe-negation
     'no-negated-in-lhs': 'off',
 
     // Disallow assignments that can lead to race conditions due to usage of await or yield
     // https://eslint.org/docs/rules/require-atomic-updates
-    // TODO: enable, semver-major
+    // note: not enabled because it is very buggy
```

Comment:
Hi, @ljharb !

Would you mind to share some cases where you faced bugs in this rule?

Reply:
There’s a couple threads on eslint core that are full of examples; i don’t have any off hand.

Reply:
right, might have guess myself to look there. thx